feature/bgp/policybase/otg_tests/comm_match_action_test/comm_match_action_test.go

```diff
@@ -0,0 +1,295 @@
+
+	pdef2 := rp.GetOrCreatePolicyDefinition("routePolicy")
+	stmt2, err := pdef2.AppendNewStatement("match_community")
+	if err != nil {
+		t.Fatalf("AppendNewStatement(%s) failed: %v", "match_community", err)
+	}
+	stmt2.GetOrCreateActions().SetPolicyResult(oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE)
+
+	communitySet := rp.GetOrCreateDefinedSets().GetOrCreateBgpDefinedSets().GetOrCreateCommunitySet(communitySetName)
+	communitySet.SetMatchSetOptions(oc.BgpPolicy_MatchSetOptionsType_ANY)
+
+	if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
+		stmt1.GetOrCreateConditions().GetOrCreateBgpConditions().SetCommunitySet(communitySetName)
+	} else {
+		stmt1.GetOrCreateConditions().GetOrCreateBgpConditions().GetOrCreateMatchCommunitySet().SetCommunitySet(communitySetName)
+	}
+
+	communitySet2 := rp.GetOrCreateDefinedSets().GetOrCreateBgpDefinedSets().GetOrCreateCommunitySet(communitySetName2)
+	communitySet2.SetMatchSetOptions(oc.BgpPolicy_MatchSetOptionsType_ANY)
+
+	if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
+		stmt2.GetOrCreateConditions().GetOrCreateBgpConditions().SetCommunitySet(communitySetName2)
+	} else {
+		stmt2.GetOrCreateConditions().GetOrCreateBgpConditions().GetOrCreateMatchCommunitySet().SetCommunitySet(communitySetName2)
+	}
+
+	gnmi.Replace(t, dut, gnmi.OC().RoutingPolicy().Config(), rp)
+
+	dni := deviations.DefaultNetworkInstance(dut)
+	pathV6 := gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().Neighbor(ipv6).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).ApplyPolicy()
+	policyV6 := root.GetOrCreateNetworkInstance(dni).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).GetOrCreateBgp().GetOrCreateNeighbor(ipv6).GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).GetOrCreateApplyPolicy()
+	policyV6.SetDefaultImportPolicy(oc.RoutingPolicy_DefaultPolicyType_REJECT_ROUTE)
+	policyV6.SetImportPolicy([]string{"routePolicy"})
+	gnmi.Replace(t, dut, pathV6.Config(), policyV6)
+
+	pathV4 := gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().Neighbor(ipv4).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).ApplyPolicy()
+	policyV4 := root.GetOrCreateNetworkInstance(dni).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).GetOrCreateBgp().GetOrCreateNeighbor(ipv4).GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).GetOrCreateApplyPolicy()
+	policyV4.SetDefaultImportPolicy(oc.RoutingPolicy_DefaultPolicyType_REJECT_ROUTE)
+	policyV4.SetImportPolicy([]string{"routePolicy"})
+	gnmi.Replace(t, dut, pathV4.Config(), policyV4)
+
+}
+
+func configureOTG(t *testing.T, bs *cfgplugins.BGPSession, prefixesV4 [][]string, prefixesV6 [][]string) {
+	var communityMembers = [][][]int{
+		{
+			{100, 1}, {200, 2}, {300, 3},
+		},
+		{
+			{100, 1},
+		},
+		{
+			{109, 1},
+		},
+		{
+			{200, 1},
+		},
+		{
+			{100, 1},
+		},
+	}
```

Comment:
This does not match the RT-7.8 readme?

https://github.com/openconfig/featureprofiles/blob/main/feature/bgp/policybase/otg_tests/comm_match_action_test/README.md?plain=1#L22-L24

Reply:
updated